spec/controllers/items_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe ItemsController, type: :controller do
+
+  describe "GET #index" do
+    it "returns http success" do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+
+    it "should assigns items as an empty array" do
+      get :index
+
+      expect(assigns[:items]).to be_empty
+    end
+    
+    context 'When given keyword' do
```

Comment:
__[rubocop]__
* Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

<!-- Comment by [SideCI](https://sideci.com) -->